### DISCUSSION TITLE
How do I download the repository archive from workflow?

### DISCUSSION THREAD

**Discussion by accountnujen** [community/community#26762] [Actions]
<p>I start the machine and I need to download the archive, which I then unzip. I can’t figure out the variables. The documentation says that it should work just through the “$” symbol, but nothing works for me.</p>
<p>My code:</p>
<pre><code class="lang-auto">runs-on: windows-latest
    timeout-minutes: 9999
 
    steps:
    - name: Download archive
      run: Invoke-WebRequest -Headers @{"Authorization" = "token $GH_TOKEN"} -Uri https://raw.githubusercontent.com/$REPO_PATH/main/down.zip -OutFile 1.zip 
      env:
        GH_TOKEN: ${{ github.token }}
        REPO_PATH: ${{ github.repository }}
    - name: Unzip Archive
      run: Expand-Archive -Path 1.Zip -DestinationPath ./
</code></pre>
<p>Please help me</p>

**Comment by Simran-B** [comment, marked as answer]
<p>In PowerShell, the way you access the environment variables is <code>$Env:GH_TOKEN</code> and <code>$Env:REPO_PATH</code>.</p>
